### ISSUE TITLE
Buttons in the "Preference" dialog are incorrectly coloured 

### ISSUE THREAD

**Issue by aplteam** [Dyalog/ride#637]
### Describe the issue you are having

While the "OK" and the "Apply" button are blue, the "Cancel" button is not. I don't think that buttons should be coloured at all, but certainly not in a way that make them look inactive.

### Did you connect to an already running interpreter or start the interpreter from RIDE?

n/a

### How do you reproduce the issue?

See above

### Paste the contents of Help → About (Shift+F1)
```
IDE:
  Version: 4.3.3463
  Platform: Linux x86_64
  Date: 2020-07-07 10:44:21 +0100
  Git commit: 0cd6e9faf39a5d6a5f5caa94d1ff51743d5cdd75
  Preferences:{
    "kbdLocale":"en_US"
  }

Interpreter:
  Version: unknown
  Platform: unknown
  Edition: unknown
  Date: unknown

```

**Comment by aplteam** [comment]
Also, with the Dark theme, when I hover with the mouse over "Cancel" it changes to colour from something I associate with "inactive" to unreadable.

**Comment by abrudz** [comment]
dup of #733

**Comment by e9gille** [comment]
Fixed in bd4b7ee1d6107f7c64534e196511dce21b8cbe5c
